dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -15,6 +15,16 @@ window.Dashboard = {
 	waitForRouteHandler: Promise.resolve(),
 
 	run: function () {
+		var self = this, loadURL, resolveWaitForNav;
+		this.waitForNav = new Promise(function(resolve, reject) {
+			resolveWaitForNav = resolve;
+		});
+
+		loadURL = function() {
+			Marbles.history.loadURL();
+			resolveWaitForNav();
```

Comment:
The issue appears to be that this is being called after `HTTPS_CERT_MISSING` event is handled.